gradle/compile.gradle

```diff
@@ -28,7 +28,9 @@ if (java7Home) {
     tasks.withType(AbstractCompile) {
         options.with {
             fork = true
-            forkOptions.javaHome = jvmForCompilation.javaHome
+            if (project.name != 'testingJunitPlatform') {
```

Comment:
It's probably better to reference the project by path to avoid potential future breakage if the project is renamed.

Reply:
should we be adding support around configuring a Java 8 home?

Reply:
@bmuschko Thanks, I've already used a property to control this behavior.
@acourtneybrown If not configured, the default java home is the same as Gradle bootstrap JDK.